babel.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (api) {
+  api.cache(true);
+
+  return {
+    presets: [
+      [
+        '@babel/preset-env',
+        {
+          targets: {
+            node: true,
```

Comment:
Node is set here because we're transforming these for Jest tests, which run in a Node environment.